docs/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/go-acme/lego/docs
+
+go 1.21
```

Comment:
```suggestion
go 1.20
```
Since go1.21, it is a hard constraint (blocking minimum go version) so it's better to use go1.20.

Reply:
Changed as requested.